app/static/js/bonnerManagement.js

```diff
@@ -14,14 +14,21 @@ function cohortRequest(year, method, username){
   })
 }
 
+function addSearchCapabilities(inputElement){
+    inputElement = $(inputElement);
```

Comment:
Nitpicky I apologize but I think this line could be removed and we just call .on() on $(inputElement).

Reply:
I like that change